src/components/dar_collection_table/ChairActions.js

```diff
@@ -101,17 +89,17 @@ export default function ChairActions(props) {
   const { showConfirmationModal, collection, relevantDatasets, goToVote } = props;
   const { dars } = collection;
   const collectionId = collection.darCollectionId;
-  const [openEnabled, setOpenEnabled] = useState(false);
-  const [cancelEnabled, setCancelEnabled] = useState(false);
-  const [voteEnabled, setVoteEnabled] = useState(false);
+  const [openEnabled, setOpenEnabled] = useState(props.openEnabled);
+  const [cancelEnabled, setCancelEnabled] = useState(props.cancelEnabled);
+  const [voteEnabled, setVoteEnabled] = useState(props.voteEnabled);
   const [voteLabel, setVoteLabel] = useState('Vote');
 
   //if there's something wrong with the collection it's best to fail gracefully
   //use this function to hide buttons on processing err
   const updateStateOnFail = () => {
     setOpenEnabled(false);
     setCancelEnabled(false);
-    setVoteLabel(false);
+    setVoteEnabled(false);
```

Comment:
unsure: do we also need to keep `setVoteLabel(false)` here?

Reply:
good catch, adding this back!